icicle/README.md

```diff
@@ -6,6 +6,7 @@
 mkdir -p build;
 cmake -DBUILD_TESTS=ON -DCURVE=<support_curve> -S . -B build;
 cmake --build build;
+./build/runner --gtest_brief=1;
```

Comment:
@vhnatyk Is this needed?

Reply:
to run the tests

Reply:
We don't need to call `ctest` anymore? Or is that the same as this?

Reply:
the same, and the binary has linked gtest main - so accepts some extra parameters like formatting of output to show only failed tests and failure file:line etc